### ISSUE TITLE
Unable to Install GTK2 or GTK3 versions on Centos 7.6

### ISSUE THREAD

**Issue by kanekennedy** [asbru-cm/asbru-cm#274]
The Perl modules are (allegedly) installed but fail to get picked up by the installer.

Any suggestions?

The GTK3 failures are:
`---> Package asbru-cm.noarch 0:6.0.0~1573378107-1.el7 will be installed
--> Processing Dependency: perl(HTTP::Proxy) for package: asbru-cm-6.0.0~1573378107-1.el7.noarch
--> Processing Dependency: perl(Gtk3::SimpleList) for package: asbru-cm-6.0.0~1573378107-1.el7.noarch
--> Finished Dependency Resolution
Error: Package: asbru-cm-6.0.0~1573378107-1.el7.noarch (asbru-cm_gtk3)
           Requires: perl(HTTP::Proxy)
Error: Package: asbru-cm-6.0.0~1573378107-1.el7.noarch (asbru-cm_gtk3)
           Requires: perl(Gtk3::SimpleList)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
[15:33:58] dev2:/tmp  cpanm Gtk3::SimpleList
Gtk3::SimpleList is up to date. (0.18)
[15:34:12] dev2:/tmp  cpanm HTTP::Proxy     
HTTP::Proxy is up to date. (0.304)
[15:34:31] dev2:/tmp `

GTK2:
`---> Package asbru-cm.noarch 0:5.2.1-1.el7 will be installed
--> Processing Dependency: perl(HTTP::Proxy) for package: asbru-cm-5.2.1-1.el7.noarch
--> Finished Dependency Resolution
Error: Package: asbru-cm-5.2.1-1.el7.noarch (asbru-cm_asbru-cm)
           Requires: perl(HTTP::Proxy)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
[15:43:48] dev2:/tmp  cpanm HTTP::Proxy
HTTP::Proxy is up to date. (0.304)`

**Comment by gfrenoy** [comment]
CentOS is not yet supported.  As you have already found out, some rpm are apparently not available anywhere :(

We are looking for help on this.  See #60 and especially the last comment from @popxunga https://github.com/asbru-cm/asbru-cm/issues/60#issuecomment-552182157

**Comment by popxunga** [comment]
@kanekennedy 
Can you add the test repo to your config (check the thread indicated in the previous post) and test ?

**Comment by gfrenoy** [comment]
I believe this is fixed by #60 now ; thanks all for your help.

From an up to date CentOS machine, simply do:

```
sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
curl -s https://packagecloud.io/install/repositories/asbru-cm/el-test/script.rpm.sh | sudo bash
curl -s https://packagecloud.io/install/repositories/asbru-cm/gtk3/script.rpm.sh | sudo bash
sudo yum install asbru-cm
```
